x-pack/plugins/fleet/server/types/models/preconfiguration.ts

```diff
@@ -36,7 +41,13 @@ export const PreconfiguredPackagesSchema = schema.arrayOf(
         }
       },
     }),
-  })
+  }),
+  {
+    defaultValue: DEFAULT_PACKAGES_NAMES.map((name) => ({
```

Comment:
this is not great behavior because if I only have system in my kibana.yml:

```
xpack.fleet.packages:
  - name: system
    version: latest
```

then none of the other required packages will be installed. can we merge the user preconfig with `DEFAULT_PACKAGES` instead?

Reply:
The config schema API doesn't support merges in default values, but we can do that downstream in the Setup function for sure.